### PR TITLE
add Other hub and update description in hub pages

### PR DESCRIPTION
**Pull request by marcochavezf**
cc @roryabraham 

### Details

Coming from this [convo](https://expensify.slack.com/archives/C02QSAC6BJ8/p1666978058174249?thread_ts=1665788470.427459&cid=C02QSAC6BJ8), we want to [remove](https://github.com/Expensify/App/pull/12248) the `Coming soon` page, but there are some changes that need to be done first. Some those changes included in this PR are:
- Add the `Other` hub in the homepage and LHN.
- Update descriptions for the hub pages (Send money, Request money and Other).
- Remove the dummy text (lorem ipsum).

### Fixed Issues

$ https://github.com/Expensify/Expensify/issues/238765

### Tests

Pre-requisites (only in dev):

In order to run the site locally you would need ruby and bundler installed in your machine, more info [here](https://github.com/Expensify/App/tree/main/docs#pre-requisites).

Then just execute the following commands in your command-line:
```
cd docs
bundle install
bundle exec jekyll serve
```

Steps:

1. Open the site (http://localhost:4000/main in dev, https://help.expensify.com/main in production)
2. Verify the Other hub is displayed on both the homepage and the LHN.
3. Navigate to the other hubs and verify the description is updated according to [these comments](https://github.com/Expensify/Expensify/issues/238867#issuecomment-1295326508).
4. Verify the navigation is working correctly between hub pages, articles, and the home page.
5. Verify there's no `lorem ipsum` text anymore in the homepage.

- [x] Verify that no errors appear in the JS console

### QA Steps

Same steps as `Test` section (test only in production).

- [ ] Verify that no errors appear in the JS console

#### PR Author Checklist
- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

<details>
<summary><h4>PR Reviewer Checklist</h4>

The reviewer will copy/paste it into a new comment and complete it after the author checklist is completed
</summary>

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [ ] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

</details>

#### Web

https://user-images.githubusercontent.com/6829422/198731585-0ee1b5e7-7f2d-478f-85af-d8c6394dd4b9.mov

#### Mobile Web - Chrome

https://user-images.githubusercontent.com/6829422/198731636-f7e64ef1-67ad-469b-8b12-a926abfd6b95.mov

#### Mobile Web - Safari

https://user-images.githubusercontent.com/6829422/198731719-912e5a46-12f0-4fa6-9b4a-53be7ba5ad8d.mov

Expesify Help only works on web.

#### Desktop

N/A

#### iOS

N/A

#### Android

N/A

### CODE REVIEW

**Comment by mananjadhav** [comment]
@marcochavezf Could you please help me with the context for this one?  Are these pages for new expensify app?

**Comment by johncschuster** [comment]
@mananjadhav These are pages for our new help site. For a bit of historical context, we've been using a community forum platform (vanillaforums) to host our help site for a few years but found the platform doesn't serve us well, so we opted to move to use GitHub Pages to host our help site articles.

Does that help? Let me know if you need any more context!

**Comment by mananjadhav** [comment]
Okay thanks @johncschuster. I’ll get to it in a few hours

**Review by grgia — APPROVED** [review]
LGTM 
Screen Shot 2022-10-31 at 9 59 24 AM
Screen Shot 2022-10-31 at 9 59 31 AM

File: docs/_sass/_main.scss [hunk 1]
```diff
@@ -402,19 +404,10 @@ button {
     font-weight: bold;
     cursor: pointer;
     color: $color-dark;
+    background-color: $color-gray1;
 
-    @include card-border($color-gray2, 1px);
```
Comment (roryabraham) [inline docs/_sass/_main.scss:406]: Why did we just get rid of the card-borders? I really liked them stylistically and couldn't find any context for that decision.

**Review by roryabraham — CHANGES_REQUESTED** [review]
Why did we just get rid of the colored card borders? I really liked them stylistically and couldn't find any context for that decision.

**Comment by roryabraham** [comment]
Oh, looks like it just came from [here](https://github.com/Expensify/Expensify/issues/238765#issue-1424335010)

**Comment by marcochavezf** [comment]
> Why did we just get rid of the colored card borders? I really liked them stylistically and couldn't find any context for that decision.

Ah yup, the design was updated in this [GH](https://github.com/Expensify/Expensify/issues/236858).

**Review by roryabraham — APPROVED** [review]
LGTM, tests well. We don't need C+ reviews on HelpDot PRs, because it's a totally separate beast and still behind beta.

**Comment by roryabraham** [comment]
Not an emergency, just ignored the NewDot checklist for this HelpDot PR. Probably shouldn't do that in the future.

**Comment by mananjadhav** [comment]
Ohh! @roryabraham I just did all the testing and was about to submit the screenshots.

Screenshot 2022-11-01 at 12 16 51 AM

Screenshot 2022-11-01 at 12 16 18 AM

**Comment by roryabraham** [comment]
No worries, we can make the payment in this case

> We don't need C+ reviews on HelpDot PRs

This isn't an "official policy", just my opinion that hasn't been widely discussed. We can issue the payment this time since you did the work.

**Comment by mananjadhav** [comment]
Okay thanks for the consideration.

**Comment by johncschuster** [comment]
Adding myself to the Assignee list to get an Upwork job put together. I'll ping you when I've got that created, @mananjadhav 👍

**Comment by mananjadhav** [comment]
@johncschuster @marcochavezf This is deployed on production for 7 days and eligible for C+ Review payment. Can someone please help with this?

**Comment by marcochavezf** [comment]
Sure, I asked [internally](https://expensify.slack.com/archives/C01SKUP7QR0/p1668211886107129) how to proceed with the payment.

**Comment by mananjadhav** [comment]
@marcochavezf Did you get anyone to help with this?

**Comment by marcochavezf** [comment]
Hey @mananjadhav, @dylanexpensify will help us to proceed with the payment 👍🏽

**Comment by mananjadhav** [comment]
Okay thanks for the update and thanks @dylanexpensify for taking this up.

**Comment by dylanexpensify** [comment]
Upwork here: https://www.upwork.com/jobs/~01902bdefab70e957b let's get you paid!!! @mananjadhav

**Comment by mananjadhav** [comment]
@dylanexpensify Applied.

**Comment by dylanexpensify** [comment]
Offer sent!

**Comment by dylanexpensify** [comment]
payment sent, job closed, contract ended

**Pull request merged**